bigtable/client/testing/inprocess_data_client.h

```diff
@@ -0,0 +1,65 @@
+// Copyright 2018 Google Inc.
```

Comment:
s/Inc./LLC/  we changed this boilerplate recently.

Reply:
fixed